docs/tutorials/geomesa-examples-authorizations.rst

```diff
@@ -454,7 +457,7 @@ Configure GeoServer for PKI Login
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Follow the instructions located
-`here <http://docs.geoserver.org/stable/en/user/security/tutorials/cert/index.html>`__
+`here <http://docs.geoserver.org/2.9.1/user/security/tutorials/cert/index.html>`__
```

Comment:
Yeah, nominally 2.9.1 is the version of GeoServer that GM 1.3.x (and thus the tutorials) supports, but GeoServer stable is already on 2.11.